test/unit/console.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Expect, Test, TestCase, IgnoreTest, FocusTest } from "alsatian";
+import * as util from "../src/util";
+
+export class ConsoleTests {
+
+    @TestCase("console.log()", "print();")
+    @TestCase('console.log("Hello")', 'print("Hello");')
+    @TestCase('console.log("Hello %s", "there")', 'print(string.format("Hello %s", "there"));')
+    @TestCase('console.log("Hello", "There")', 'print("Hello", "There");')
+    @TestCase("console.trace()", "print(debug.traceback());")
+    @TestCase('console.trace("message")', 'print(debug.traceback("message"));')
+    @TestCase('console.trace("Hello %s", "there")', 'print(debug.traceback(string.format("Hello %s", "there")));')
+    @TestCase('console.trace("Hello", "there")', 'print(debug.traceback("Hello", "there"));')
+    @TestCase("console.assert(false)", "assert(false);")
+    @TestCase('console.assert(false, "message")', 'assert(false, "message");')
+    @TestCase('console.assert(false, "message %s", "info")', 'assert(false, string.format("message %s", "info"));')
+    @TestCase('console.assert(false, "message", "more")', 'assert(false, "message", "more");')
+    @Test("Console")
```

Comment:
You can split these up into different test cases: one for log, one for trace, one for assert, and put the input parameters in the TestCase (i.e. `@TestCase("")`, `@TestCase("Hello %")`)